configure.ac

```diff
@@ -69,6 +69,13 @@ x86_64*)
   ;;
   esac
 ;;
+*64*)
+;;
+*)
+AC_MSG_WARN([
+*** __BIGREGION__ is not defined.
+*** Adjust the address limits in sasconf.h as required.])
+;;
```

Comment:
@tuliom Yes it is disabled by default for 32 bit. Just added a warning to inform that if the address limits differ, sasconf change would be required.